compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -3050,6 +3124,8 @@ declare_lint_pass! {
         BREAK_WITH_LABEL_AND_LOOP,
         UNUSED_ATTRIBUTES,
         NON_EXHAUSTIVE_OMITTED_PATTERNS,
+        INVALID_CFG_NAME,
+        INVALID_CFG_VALUE,
```

Comment:
We have some naming conventions for lints - https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#lints.
Under those conventions the lint names would be `invalid_cfg_names` and `invalid_cfg_values`.

I would also recommend changing the lint names to `unknown_cfg_names` and `unknown_cfg_values` (or to `unexpected_cfg_names` and `unexpected_cfg_values`), because the cfg predicates are valid, they are just suspicious because they are not expected.

Reply:
I'm also not sure that two separate lints are necessary here, and under what circumstances we'd need to discern between them in code.
They are checking for basically the same thing - a suspicious `cfg` predicate, I think it would be better to not multiply entities and merge them into a single lint.
cc @sivadeilra